tools/downloader/topologies_test.py

```diff
@@ -20,13 +20,75 @@ def test_get_ir(self):
         self.assertEqual(os.path.basename(bin_path), 'mobilenet-ssd.bin')
 
     def test_get_intel_ir(self):
-        action_recognition = omz.load_model('face-detection-0100', 'FP16')
-        xml_path, bin_path = action_recognition.get_ir()
+        face_detection = omz.load_model('face-detection-0100', 'FP32')
+        # xml_path, bin_path = face_detection.get_ir()
+
+        # print(xml_path)
+
+        # self.assertTrue(os.path.exists(xml_path))
+        # self.assertTrue(os.path.exists(bin_path))
+        # self.assertEqual(os.path.basename(xml_path), 'face-detection-0100.xml')
+        # self.assertEqual(os.path.basename(bin_path), 'face-detection-0100.bin')
+
+    def test_load_public_without_precision_and_get_ir(self):
+        ssd = omz.load_model('mobilenet-ssd')
+        xml_path, bin_path = ssd.get_ir()
+        xml_path_16, bin_path_16 = ssd.get_ir('FP16')
+        xml_path_32, bin_path_32 = ssd.get_ir('FP32')
+
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+        self.assertTrue(os.path.exists(xml_path_16))
+        self.assertTrue(os.path.exists(bin_path_16))
+        self.assertTrue(os.path.exists(xml_path_32))
+        self.assertTrue(os.path.exists(bin_path_32))
+
+        self.assertEqual(os.path.basename(xml_path), 'mobilenet-ssd.xml')
+        self.assertEqual(os.path.basename(bin_path), 'mobilenet-ssd.bin')
+        self.assertEqual(os.path.basename(xml_path_16), 'mobilenet-ssd.xml')
+        self.assertEqual(os.path.basename(bin_path_16), 'mobilenet-ssd.bin')
+        self.assertEqual(os.path.basename(xml_path_32), 'mobilenet-ssd.xml')
+        self.assertEqual(os.path.basename(bin_path_32), 'mobilenet-ssd.bin')
```

Comment:
Please format like this:
```python
ssd = omz.load_model('mobilenet-ssd')
for p in [None, 'FP32', 'FP16']:
  xml_path, bin_path = ssd.get_ir(precision=p)
  
  self.assertTrue(os.path.exists(xml_path))
  self.assertTrue(os.path.exists(bin_path))
  // also add a check for parent folder - FP32 for None and FP32 and FP16 for FP16
```

Reply:
Even better to parametrize the test so it will accept downloading precision and `get_ir` precision and check final precision as

| ↓ load / get_ir → | None | FP32 | FP16 |
|---|---|---|---|
| None | FP32 | FP32 | FP16 |
| FP32 | FP32 | FP32 | FP16 |
| FP16 | FP16 | FP32 | FP16 |

```python
final = get_ir_precision if get_ir_precision else download_precision
```